tapchannel/aux_invoice_manager.go

```diff
@@ -419,6 +419,40 @@ func IsAssetInvoice(invoice *lnrpc.Invoice, rfqLookup RfqLookup) bool {
 	return false
 }
 
+// GetBuyQuoteFromRouteHints is a helper method that extracts a buy quote from
+// the route hints of an invoice, given that the quote is accepted and has a
+// matching specifier. If no matching quote is found, it returns an error.
+func (s *AuxInvoiceManager) GetBuyQuoteFromRouteHints(invoice *lnrpc.Invoice,
+	specifier asset.Specifier) (*rfqmsg.BuyAccept, error) {
+
+	hints := invoice.RouteHints
+	buyQuotes := s.cfg.RfqManager.PeerAcceptedBuyQuotes()
+
+	for _, hint := range hints {
```

Comment:
nit: for better readability consider the following:
```
for _, routeHint := range invoice.RouteHints {
    for _, hopHint := range routeHint.HopHints {
```

Reply:
Agree, fixed.